pom.xml

```diff
@@ -62,7 +62,7 @@
         <!-- Must MANUALLY update this if the codice-maven project's version changes -->
         <codice-maven.version>0.2</codice-maven.version>
         <!-- Must MANUALLY update this if the codice-test project's version changes -->
-        <codice-test.version>0.3</codice-test.version>
+        <codice-test.version>0.10-SNAPSHOT</codice-test.version>
```

Comment:
❗️ Not sure we should be referencing a snapshot version. Can we use 0.9 or release 0.10?